app/Resources/FOSUserBundle/views/layout.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}
+    {{ 'add translation here' }}
+{% endblock %}
+
+{% block body %}
+    <br>
```

Comment:
Maybe give some distance to header. I'm not really sure what you want to achieve and where you want it. We can talk about it. It's ok for now.